specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/cloud-events-schema/email_delivery_report_received.json

```diff
@@ -0,0 +1,15 @@
+{
+  "eventId": "f7c02d97-069a-4f60-9e30-e49939575db3",
+  "eventTime": "2023-01-20T15:57:08.2653107-08:00",
+  "dataVersion": "1.0",
+  "acsResourceId": "0a420b29-555c-4f6b-841e-de8059893bb9",
+  "recipient": "test1@contoso.com",
+  "sender": "test2@contoso.org",
+  "deliveryAttemptTimeStamp": "2023-01-20T23:57:08.2662144+00:00",
+  "userActionTimeStamp": "2023-01-20T23:57:08.2677196+00:00",
+  "messageId": "8c339faf-e8e0-4437-9dae-792b2f6aee09",
+  "status": "delivered",
+  "engagementContext": "Link clicked",
```

Comment:
This should contain a URL.

Reply:
This should contain the cloud event envelope.

Reply:
update with correct sample